packages/Popover/tests/popover.cypress.js

```diff
@@ -20,6 +20,15 @@ describe("Popover", () => {
     cy.get(selectors.popover).should("be.not.visible");
   });
 
+  it("should still be visible when click inside", () => {
```

Comment:
```suggestion
  it("should be visible when clicked inside popover", () => {
```
or `should be visible when clicked inside`. I think we can cut out the `still`